gulpfile.js

```diff
@@ -263,12 +263,14 @@ gulp.task('test', function () {
 gulp.task('watch', ['build:development'], function () {
   var jsWatcher = gulp.watch([ assetsFolder + '/**/*.js' ], ['js']);
   var cssWatcher = gulp.watch([ assetsFolder + '/**/*.scss' ], ['sass']);
+  var frameworkWatcher = gulp.watch([ bowerRoot + '/digitalmarketplace-frameworks' + '/**/*.yml' ], ['copy:frameworks']);
```

Comment:
We should probably use `sspContentRoot` here in case the repository name changes again in the future